tests/FixerFactoryTest.php

```diff
@@ -380,15 +379,38 @@ public function testFixerDefinitions(FixerInterface $fixer)
             $this->markTestIncomplete('ShortFixerDefinition does not contains all needed information.');
         }
 
-        $this->assertNotEmpty($definition->getCodeSamples(), 'Code samples are required.');
+        $samples = $definition->getCodeSamples();
+        $this->assertInternalType('array', $samples, 'Code samples must be an array.');
```

Comment:
for now this is a nice DX when adding the descriptions, I see no harm in keeping this.